src/enclave/enclave_mem.c

```diff
@@ -390,7 +270,7 @@ void* enclave_mmap(
         if (prot != -1 && (!zero_pages || found_only_fresh_pages))
         {
             // Set requested page permission
-            mprotect(ret, length, prot);
+			sgxlkl_host_syscall_mprotect(&mprotect_ret, ret, length, prot);
```

Comment:
Is there some weird indentation here?

Reply:
I forgot to run clang-format before I committed this.  Fixed now.